src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -187,6 +211,211 @@ class UserService(protected val userInfo: UserInfo, dataSource: DataSource, prot
     }
   }
 
+  def listAdmins() = {
+    asAdmin {
+      gcsDAO.listAdmins.map(RequestComplete(StatusCodes.OK, _)).recover{ case throwable =>
```

Comment:
I'm not seeing any changes to WorkplaceService as part of this PR.  Do you need to commit those?